.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "4"
```

Comment:
it should be `4`
https://github.com/ember-cli/ember-addon-output/blob/v3.0.0/.travis.yml